diff --git a/node_modules/rn-fetch-blob/ios/RNFetchBlob/RNFetchBlob.h b/node_modules/rn-fetch-blob/ios/RNFetchBlob/RNFetchBlob.h
index 669a093..3618600 100644
--- a/node_modules/rn-fetch-blob/ios/RNFetchBlob/RNFetchBlob.h
+++ b/node_modules/rn-fetch-blob/ios/RNFetchBlob/RNFetchBlob.h
@@ -4,35 +4,27 @@
 //  Created by wkh237 on 2016/4/28.
 //
 
-//XXX: DO NO REMOVE THIS LINE IF YOU'RE USING IT ON RN > 0.40 PROJECT
-
-
-
 #ifndef RNFetchBlob_h
 #define RNFetchBlob_h
 
-
 #if __has_include(<React/RCTAssert.h>)
 #import <React/RCTLog.h>
 #import <React/RCTRootView.h>
 #import <React/RCTBridge.h>
-#import <React/RCTEventDispatcher.h>
 #import <React/RCTBridgeModule.h>
+#import <React/RCTEventEmitter.h>   // ✅ Replace RCTEventDispatcher
 #else
 #import "RCTBridgeModule.h"
 #import "RCTLog.h"
 #import "RCTRootView.h"
 #import "RCTBridge.h"
-#import "RCTEventDispatcher.h"
+#import "RCTEventEmitter.h"         // ✅ Replace RCTEventDispatcher
 #endif
 
 #import <UIKit/UIKit.h>
 
-
 @interface RNFetchBlob : NSObject <RCTBridgeModule, UIDocumentInteractionControllerDelegate> {
-
     NSString * filePathPrefix;
-
 }
 
 @property (nonatomic) NSString * filePathPrefix;
@@ -43,3 +35,4 @@
 @end
 
 #endif /* RNFetchBlob_h */
+
